regex2tree.erl

```diff
@@ -0,0 +1,76 @@
+-module(regex2tree).
+-include_lib("eunit/include/eunit.hrl").
+
+%% API
+-export([re2post/1,
+         extract_from_parentheses/1]).
+
+re2post([]) -> [];
+re2post(Input) -> re2post(Input, 0, [], []).
+
+re2post([], WrittenCount, OperatorStack, Output) ->
```

Comment:
is it operator tho? Aren't these for character literals?

Reply:
It is indeed only for Operators, more precisely only binary operators are put there (which only the `+` for now)